client/components/ratings/data-breakdown.jsx

```diff
@@ -4,25 +4,8 @@ import sampleObj from '../Sample_data/SampleMetadata.js';
 import StarGraph from './StarGraph.jsx';
 import Characteristics from './characteristics.jsx';
 
-class Breakdown extends React.Component {
-  constructor (props) {
-    super(props);
-
-    this.state = {
-      productId: 0
-    }
-
-    this.transformData = this.transformData.bind(this);
-
-  }
-
-  getProductMetadata(product_id) {
-    axios.get('/products/metadata')
-      .then( () => console.log(data))
-      .catch( () => console.log(err));
-  }
-
-  transformData(dataObj) {
+var Breakdown = (props) => {
+  const transformData = function(dataObj) {
```

Comment:
Is the function closing bracket missing? I cannot see it.

Reply:
It's there or else the program would've broken